sweep/fee_bumper.go

```diff
@@ -902,34 +902,28 @@ func (t *TxPublisher) processRecords() {
 	// For records that are confirmed, we'll notify the caller about this
 	// result.
 	for requestID, r := range confirmedRecords {
-		rec := r
```

Comment:
yes - and more importantly we now check the go version when users build it themselves so the loopvar is guaranteed. also because of [this](https://github.com/lightningnetwork/lnd/pull/8893#discussion_r1689638928).